backend/applications/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import include, path
+from rest_framework import routers
+
+from applications.api.v1 import views
+
+app_name = "applications"
+
+router = routers.DefaultRouter()
```

Comment:
Just a note that there should be only one `DefaultRouter` per API, so you will probably run into problems with this if you add other applications than this `applications` (HEHE) to the API.